bitrise.yml

```diff
@@ -0,0 +1,437 @@
+format_version: "4"
```

Comment:
This file need to be removed from the VSC. It contain's password's which should never be committed.

Reply:
Removed.

Reply:
Looks like still not removed

Reply:
Ok I have fixed it

Reply:
k. I already changed the exposed env var's. Keep in mind that everything committed and pushed stay's forever in git.